samples/attested_tls/server/host/host.cpp

```diff
@@ -43,15 +43,30 @@ int main(int argc, const char* argv[])
     oe_result_t result = OE_OK;
     int ret = 1;
     char* server_port = NULL;
+    bool keep_server_up = false;
 
     /* Check argument count */
     if (argc != 3)
     {
+        if (argc == 4)
+        {
+            char* loop_option = (char*)"-server-in-loop";
```

Comment:
why is this char* and not const char*?   strncmp should accept const char* on  Linux, Windows, and anything else that conforms to the posix standard.
```suggestion
            const char* loop_option = "-server-in-loop";
```

You could also just #define LOOP_OPTION "-server-in-loop" and use LOOP_OPTION in the code below.